packages/fast-tooling-react/src/form/templates/template.control.standard.props.ts

```diff
@@ -0,0 +1,23 @@
+import {
+    AbstractControlTemplateProps,
+    ControlConfig,
+} from "./template.control.abstract.props";
+
+export enum ControlContext {
```

Comment:
Maybe add a comment about what this is? It's not super meaningful to me as it's named now